core/src/main/java/org/apache/druid/java/util/common/StringUtils.java

```diff
@@ -481,9 +482,10 @@ public static String repeat(String s, int count)
    *
    * @return the string left-padded with pad to a length of len
    */
-  public static String lpad(String base, Integer len, String pad)
+  @Nullable
+  public static String lpad(@Nonnull String base, int len, @Nonnull String pad)
```

Comment:
Afaik we don't traditionally annotate things explicitly with `@Nonnull`, in preference of using `@Nullable` and `@EverythingIsNonnullByDefault` in package.info, however I guess we haven't gone through and set that everywhere. However, to be more consistent with other places in the code maybe that would be better?

Reply:
To add to this, there are plenty of existing `@Nonnull` annotations in the code, so this change isn't really required, mostly just expressing my preference to keep things from getting too verbose 😅